### PR TITLE
Adding CDEP example

### PR DESCRIPTION
**Pull request by ggfan**
simple app cut out from hello-libs to showcase the [CDEP](http://github.com/google/cdep)
@jomof  PTAL, thx

### CODE REVIEW

File: hello-cdep/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+Hello-libs
+=========
+hello-cdep is an Android sample that demos [CDEP](https://github.com/google/cdep) native lib distribution system usage.
```
Comment (jomof) [inline hello-cdep/README.md]: Title says "Hello-libs" but description says "Hello-cdep". Seems like the latter is better.

Comment (jomof) [inline hello-cdep/README.md]: I've been using "CDep" everywhere instead of "CDEP"

File: hello-cdep/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+Refer to [CDEP documentation](https://github.com/google/cdep/tree/master/doc)
```
Comment (jomof) [inline hello-cdep/README.md]: Could you change the CDep documentation linke to (https://github.com/google/cdep/blob/master/README.md)?

File: hello-cdep/cdep.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+builders: [cmake]
+
+dependencies:
+# Uncomment next line to get header-only boost
```
Comment (jomof) [inline hello-cdep/cdep.yml]: Maybe remove "# Uncomment next line"

File: hello-cdep/README.md (hunk 1 above)
Comment (ggfan) [inline hello-cdep/README.md]: done

**Comment by ggfan** [comment]
Runs on Pixel fine, merging it in!

**Pull request merged**